src/gluonts/nursery/sagemaker_sdk/docker_images/cpu_serving/Dockerfile

```diff
@@ -0,0 +1,98 @@
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon ML & Forecasting"
+
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG MMS_VERSION=1.0.5
+ARG MX_VERSION=1.4.1
+ARG PYTHON=python3
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.6.8
+
+ARG REQUIREMENTS=requirements.txt
+ARG REQUIREMENTS_EXTRAS_R=requirements-extras-r.txt
+ARG REQUIREMENTS_EXTRAS_PROPHET=requirements-extras-prophet.txt
```

Comment:
Why are these `ARG`s?

Reply:
This was before the makefile did everything, I wanted to allow for flexibility of the location. Now it makes the Dockerfile more readable because the outside dependencies are listed in the beginning clearly, but its not a necessity. I thought about doing the same for the mxnet container tar.gz too though.